### ISSUE TITLE
RFE: exclude directories to scan

### ISSUE THREAD

**Issue by glensc** [ka7/misspell_fixer#8]
feature request to exclude directories to scan. for example vendor/ dir

**Comment by ka7** [comment]
right. I've seen many examples in real world projects. ( all the jquerys, RFC's, .. ) fixing those makes no sense, just costs time. ( better to fix that in upstream and hope for future )

looks like some "find -prune" magic needed.

**Comment by glensc** [comment]
yeah. i work-arounded in my projects using clean git checkout, but it's annoying to do.
